### ISSUE TITLE
AppImage 0.1.6-beta "Error in the format of the program file" and wrong filename

### ISSUE THREAD

**Issue by WLBI** [ubports/ubports-installer#24]
I tried downloading and executing the 0.1.6 beta here:
https://github.com/ubports/ubports-installer/releases

the filename doesn't go together with the version number.
should be 0.1.6-beta and the filename contains 0.1.5-beta
When executing the file I get following error message:
**German:** 
bash: ./ubports-installer-0.1.5-beta-x86_64.AppImage: Kann die Binärdatei nicht ausführen: Fehler im Format der Programmdatei

**the error message translated to english:** 
bash: ./ubports-installer-0.1.5-beta-x86_64.AppImage: Can not execute the binary file: Error in the format of the program file

**Comment by garro95** [comment]
The error message in English is:
```
bash: ./ubports-installer-0.1.6-beta-x86_64.AppImage: cannot execute binary file: Exec format error
```

**Comment by mariogrip** [comment]
This should be fixed with 0.1.7-beta
